client/client.go

```diff
@@ -43,12 +49,36 @@ func NewClient(url, user, password string, verifyTLS bool, caCertByte []byte) *C
 		password:  password,
 		gorequest: gorequest.New().TLSClientConfig(tlsConfig),
 	}
+
+	if strings.Contains(url, "cloud.aquasec.com") {
```

Comment:
@yossig-aquasec this doesn't resolve the issue my PR was looking to fix. cloud.aquasec.com hosts both ESE and USE environments, ESE environments should be using the CSP authentication func